content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -827,6 +891,142 @@ Learn more about this on the [Configuring your Prisma Client API](/concepts/comp
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'cockroachdb']}>
+
+As a next step, you will introspect your database. The result of the introspection will be a [data model](/concepts/components/prisma-schema/data-model) inside your Prisma schema.
+
+Run the following command to introspect your database:
+
+```terminal copy
+npx prisma db pull
+```
+
+This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a Prisma data model.
+
+After the introspection is complete, your Prisma schema file was updated:
+
+![Introspect your database with Prisma](https://res.cloudinary.com/prismaio/image/upload/v1628761155/docs/f7itiYw.png)
+
+The data model now looks similar to this:
+
+```prisma file=prisma/schema.prisma
+model Post {
+  id        BigInt   @id(map: "primary") @default(autoincrement())
+  title     String   @db.VarChar(255)
+  createdAt DateTime @default(now()) @db.Timestamp(6)
+  content   String?
+  published Boolean  @default(false)
+  authorId  BigInt
+  User      User     @relation(fields: [authorId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "fk_authorId_ref_User")
```

Comment:
this might indicate a difference in default referential action values

Reply:
plus naming of foreign keys

Reply:
Naming = This is a current limitation of CockroachDB 21.1 and will go away with 22.1. See internal discussion https://prisma-company.slack.com/archives/C01CSQXTNJ3/p1642775881029100 (Will add to limitations table in connector page via suggestion)

Referential Actions = intended, actually outdated in PostgreSQL: https://github.com/prisma/docs/issues/2752

Reply:
Added the foreign keys limitation to the connector page.